.gitignore

```diff
@@ -18,3 +18,6 @@ ENV*
 node_modules
 package.json
 yarn.lock
+
+# Visual Studio Code
+.vscode
```

Comment:
I don't mind keeping this, but FYI editor specific files should usually be added in your own global `.gitignore`:

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Thanks - wasn't aware